docs/releases/v2019.1.rst

```diff
@@ -15,6 +15,7 @@ possible.
 With Gluon v2019.1, nodes will not answer respondd queries on ``[ff02::2:1001]:1001`` anymore. Respondd
 querier setups still using this address must be updated to the new address ``[ff05::2:1001]:1001``
 (supported since Gluon v2017.1). This change was required due to cross-domain leakage of respondd data.
+If you are using hopglass-server to query respondd data, you need to update it to at least commit f0e2c0a5.
```

Comment:
there is no such commit f0e2c0a5

Reply:
> there is no such commit f0e2c0a5

Yes there is: https://github.com/hopglass/hopglass-server/commit/f0e2c0a58b8947176f7c87680c2bcb27f302f14c

> github history only shows 7 characters

So what?

Reply:
@petabyteboy Can you maybe create a tag including the fix?

Reply:
Yes, I will create a 1.0.0 tag including this commit soon.

Reply:
@petabyteboy Ping.